src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -9,6 +9,8 @@ void npe(String a, String b) {
   } else {
 
   }
+  if (a == null)
+   a.charAt(123);
```

Comment:
asads

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1116723583>*